frontend/post/postDetailsDirective.js

```diff
@@ -132,7 +132,10 @@
         };
 
         postDetailsCtrl.showCommentInput = function showCommentInput() {
-            return (postDetailsCtrl.showComments || postDetailsCtrl.isPostPage) && !postDetailsCtrl.isDeleted(postDetailsCtrl.post) && !postDetailsCtrl.isInstInactive();
+            return (postDetailsCtrl.showComments || postDetailsCtrl.isPostPage) && 
+                !postDetailsCtrl.isDeleted(postDetailsCtrl.post) &&
+                !postDetailsCtrl.isInstInactive() && 
+                !postDetailsCtrl.showSurvey();
```

Comment:
why added the `showSurvey` ?

Reply:
Because the function showComments was being called along with the function showSurvey, in the template. So I decided to move the last one from template to the function showComments.